src/sage/schemes/curves/projective_curve.py

```diff
@@ -2388,15 +2429,15 @@ def _function_field(self):
         return self._open_affine._function_field
 
     @lazy_attribute
-    def _lift_to_function_field(self):
+    def _map_to_function_field(self):
         """
         Return the map to function field of the curve.
```

Comment:
Add "to _the_ function field", as you did above?

Reply:
Yes. Thanks.